src/fielddef.ts

```diff
@@ -92,9 +92,13 @@ export interface OrderFieldDef extends FieldDef {
   sort?: SortOrder;
 }
 
+export interface TextFieldDef extends FieldDef {
+  format?: string;
```

Comment:
/**
   * The formatting pattern for text value. If not defined, this will be determined automatically.
   */
Do I just add this on top of it?